start/src/main/java/org/apache/accumulo/start/classloader/vfs/AccumuloVFSClassLoader.java

```diff
@@ -283,16 +283,27 @@ private static File computeTopCacheDir() {
     void print(String s);
   }
 
-  public static void printClassPath() {
-    printClassPath(new Printer() {
-      @Override
-      public void print(String s) {
-        System.out.println(s);
-      }
-    });
+  public static void printClassPath(boolean debug) {
+    printClassPath(System.out::print, debug);
+  }
+
+  public static String getClassPath(boolean debug) {
+    StringBuilder cp = new StringBuilder();
+    printClassPath(s -> cp.append(s), debug);
+    return cp.toString();
   }
 
-  public static void printClassPath(Printer out) {
+  private static void printJar(Printer out, String jarPath, boolean debug, boolean sawFirst) {
```

Comment:
Have you verified that this still prints directories on the class path correctly? Also, do we want this class to always print the full classpath, including what's on the VFS classloader path (which might have URLs not parse-able by other class loaders). Might want to have an option for depth, with the default just being the value of `System.getProperty("java.class.path")`.